### PR TITLE
Add blue CHECK24 elePHPant

### PR DESCRIPTION
**Pull request by cookieguru**
@movetodevnull do you have photos or a name?

### CODE REVIEW

**Comment by sebastianfeldmann** [comment]
Name ist still in discussion :)
I'll attach some pictures tomorrow.

**Comment by cookieguru** [comment]
Gave you write access to my repo if you want to update the PR directly

**Comment by sebastianfeldmann** [comment]
Sweet, thanks :)

**Comment by sebastianfeldmann** [comment]
Sorry it took so long, but naming and getting the internal orders on the way took some time.
After that I had the brass to go on vacation for a couple of days :P

File: source/_elephpants/2019-07-04-blue-check24.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+categories:
+    - check24
+tags:
+    - blue
+sponsor: CHECK24
+reverse: CHECK24 logo
+photos:
+    -
+        file: blue-check24-0-400x300.jpg
+        link: https://twitter.com/CHECK24_JePHPry
```
Comment (philipsharp) [inline source/_elephpants/2019-07-04-blue-check24.md]: These links should be to the source of the photo, not a generic link for the elephpant (which is include below).

File: app/config/sculpin_site.yml [hunk 2]
```diff
@@ -16,6 +16,9 @@ subspecies:
     cakephp:
         common: "CakePHP"
         latin: "pistrini"
+    check24:
+        common: "CHECK24"
+        latin: "reprehendovigintiquattuor"
```
Comment (philipsharp) [inline app/config/sculpin_site.yml]: That's a mouthful. 😃

Comment (cookieguru) [reply app/config/sculpin_site.yml]: An alternative would be `confirmixxiv`

Comment (sebastianfeldmann) [reply app/config/sculpin_site.yml]: Fixed the "latin" version ;)

File: source/_elephpants/2019-07-04-blue-check24.md (hunk 1 above)
Comment (sebastianfeldmann) [reply source/_elephpants/2019-07-04-blue-check24.md]: I used the twitter image links but I also added the images in the photos directory. Should I leave them there or should I remove them?

File: app/config/sculpin_site.yml [hunk 3]
```diff
@@ -16,6 +16,9 @@ subspecies:
     cakephp:
         common: "CakePHP"
         latin: "pistrini"
+    check24:
+        common: "CHECK24"
+        latin: "collatio XXIV"
```
Comment (philipsharp) [inline app/config/sculpin_site.yml]: The proper form here would be `collationis`, although the number afterwards is weird. What if we drop the number? They don't have to be direct translations. A little subtly and humor is good.

Comment (sebastianfeldmann) [reply app/config/sculpin_site.yml]: Done ;)

**Comment by sebastianfeldmann** [comment]
🎉 thanks ;)

**Pull request merged**